tap-agent/src/agent/sender_account.rs

```diff
@@ -673,10 +677,13 @@ impl Actor for SenderAccount {
                 };
                 // In case we fail, we want our actor to keep running
                 if let Err(err) = rav_result {
+                    state.backoff_info.fail();
                     tracing::error!(
                         error = %err,
                         "There was an error while requesting a RAV."
                     );
+                } else {
+                    state.backoff_info.ok();
                 }
```

Comment:
This is not the place where we want our backoff to happen.

In general, the backoff.fail() should be on line 214, inside the ok_or for heaviest_allocation